src/dev/flang/tools/docs/Html.java

```diff
@@ -287,7 +287,7 @@ private String annotateAbstract(AbstractFeature af)
    */
   private String annotatePrivateConstructor(AbstractFeature af)
   {
-    return af.visibility() == Visi.PRIVPUB
+    return af.visibility().eraseTypeVisibility() != Visi.PUB && af.visibility().typeVisibility() == Visi.PUB
```

Comment:
I don't understand this logic? imho this should just be `af.visibility().eraseTypeVisibility() != Visi.PUB`?

Reply:
I guess the second part is not needed, since features with a non public type should not appear in the docs at all.